lib/qif2json.js

```diff
@@ -5,19 +5,18 @@
  * Copyright (c) 2012 Steve Mason
  * Licensed under the MIT license.
  */
-
-
 const fs = require('fs');
 const jschardet = require('jschardet');
 const Iconv = require('iconv-lite');
 const fecha = require('fecha');
 const debug = require('debug')('qif2json');
+const { TYPES } = require('./types');
 
 const US_DATE_FORMATS = ['MM-DD-YYYYHH:mm:ss', 'MM-DD-YYYY', 'MM-DD-YY'];
 const UK_DATE_FORMATS = ['DD-MM-YYYYHH:mm:ss', 'DD-MM-YYYY', 'DD-MM-YY'];
 
 function parseDate(dateStr, formats) {
-  if (formats === 'us') {
+  if (formats === 'us' || dateStr.includes("'")) {
```

Comment:
It seems a bit risky to say that any `dateStr` that includes a `'` is a US date format

Reply:
Is there any better proposition? This code do not breaks compatibility but is open also for other date format, that I am using.

Reply:
Per https://github.com/spmason/qif2json/pull/26 you can pass in a `dateFormat` option to control how the dates are parsed. `us` is only supported for backward compatibility